src/utils/allowedHTML.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Defines the list of allowed HTML tags that do not count towards the total
+ * character limit.
+ *
+ */
+const regrexTests = [
+  new RegExp("(<iframe.*/iframe>)", "gm"),
+  new RegExp("(<div.*/div>)", "gm"),
+]
+
+/**
+ * @param text raw text input by user
+ * @returns the character length after removing specifc HTML tags
+ */
+export const getLengthWithoutTags = (text: string): number => {
+  let finalText = text
+  for (const regrexTest of regrexTests) {
+    finalText = finalText.replace(regrexTest, "")
+  }
+  return finalText.length
```

Comment:
hmm, we might want to express this as a `reduce` operation if it's cleaner but i'll leave it up to you!